app/src/main/java/com/example/testoauth/ui/login/SignInActivity.java

```diff
@@ -151,6 +153,25 @@ public void onRequestSuccess(final int requestCode, @Nullable VerificationDataBu
                         Toast.LENGTH_SHORT).show();
                 showLayout(PROFILE_LAYOUT);
                 findViewById(R.id.btnVerify).setOnClickListener(verifyClickListener);
+            } else if (requestCode == VerificationCallback.TYPE_IM_OTP_INITIATED) {
+                verificationCallbackType = VerificationCallback.TYPE_IM_OTP_INITIATED;
+                String ttl = bundle.getString(VerificationDataBundle.KEY_TTL);
+                if (ttl != null) {
+                    Toast.makeText(SignInActivity.this,
+                            "IM OTP initiated with TTL : " + bundle.getString(VerificationDataBundle.KEY_TTL),
```

Comment:
We are adding values to strings folder in some places, and not in other places.
We should. not hardcode strings here